appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/__tests__/view-and-review.test.js

```diff
@@ -205,4 +213,239 @@ describe('interested-party-comments', () => {
 			expect(innerHtml).toMatchSnapshot();
 		});
 	});
+
+	describe('GET /manage-documents/:folderId/', () => {
+		beforeEach(() => {
+			nock('http://test/').get('/appeals/2/reps/5').reply(200, interestedPartyCommentForReview);
+
+			nock('http://test/')
+				.get('/appeals/2/document-folders/1')
+				.reply(200, costsFolderInfoAppellantApplication)
+				.persist();
+
+			nock('http://test/').get('/appeals/2/documents/1').reply(200, documentFileInfo);
+		});
+
+		it('should render a 404 error page if the folderId is invalid', async () => {
+			nock('http://test/').get('/appeals/2/document-folders/99').reply(404);
+
+			const response = await request.get(
+				`${baseUrl}/2/interested-party-comments/5/manage-documents/99`
+			);
+
+			expect(response.statusCode).toBe(404);
+
+			const element = parseHtml(response.text);
+			expect(element.innerHTML).toMatchSnapshot();
+
+			const unprettifiedElement = parseHtml(response.text, { skipPrettyPrint: true });
+
+			expect(unprettifiedElement.innerHTML).toContain('Page not found</h1>');
+		});
+
+		it('should render review comment page with the provided comment details', async () => {
```

Comment:
Is this test title correct? It looks like the test is rendering a manage folder page